### PR TITLE
Radar chart update v3.0.0.0

### PR DESCRIPTION
**Pull request by kullJul**

### CODE REVIEW

File: src/radarChart.ts [hunk 1]
```diff
@@ -832,16 +867,16 @@ export class RadarChart implements IVisual {
...
+    private drawAxesLabels(values: RadarChartLabel[]): void {
+        const labelSettings: LabelsSettingsCard = this.formattingSettings.labels;
 
-        let selectionLabelText: d3.Selection<d3.BaseType, RadarChartLabel, any, any> = this.mainGroupElement
+        const selectionLabelText: Selection<RadarChartLabel> = this.mainGroupElement
             .select(RadarChart.AxisSelector.selectorName)
             .selectAll(RadarChart.AxisLabelSelector.selectorName);
 
-        let filteredData: RadarChartLabel[] = values.filter((label: RadarChartLabel) => labelSettings.show && !label.hide);
+        const filteredData: RadarChartLabel[] = values.filter((label: RadarChartLabel) => labelSettings.topLevelSlice.value && !label.hide);
```
Comment (MulyukovAidar) [inline src/radarChart.ts:877]: replace topLevelSlice

File: src/settings.ts [hunk 2]
```diff
@@ -23,46 +23,318 @@
...
+    displayNameKey?: string = "Visual_Text"
+    slices: FormattingSettingsSlice[] = [this.font, this.labelColor, this.positionDropdown];
 }
 
-export class LabelSettings {
-    public show: boolean = true;
-    public color: string = "#000";
-    public fontSize: number = 8;
+export class LegendSettingsCard extends FormattingSettingsCompositeCard {
+    topLevelSlice = new formattingSettings.ToggleSwitch({
```
Comment (MulyukovAidar) [inline src/settings.ts:153]: fix

File: src/settings.ts [hunk 3]
```diff
@@ -23,46 +23,318 @@
...
+    }
+
+    public setVisibilityOfColorSlices(colorHelper: ColorHelper): void {
+        const isVisible: boolean = !colorHelper.isHighContrast;
+        this.dataPoint.visible = isVisible;
+        this.labels.color.visible = isVisible;
+        this.legend.text.labelColor.visible = isVisible;
+    }
+
+    public setMinMaxValuesForDisplay(minValue: number): void {
```
Comment (MulyukovAidar) [inline src/settings.ts:312]: maybe this needs to be refactored

File: test/visualTest.ts [hunk 4]
```diff
@@ -144,23 +218,23 @@ describe("RadarChart", () => {
             it("show", () => {
                 (dataView.metadata.objects as any).legend.show = true;
                 visualBuilder.updateFlushAllD3Transitions(dataView);
-                expect(visualBuilder.legendGroup.children()).toBeInDOM();
+                visualBuilder.legendGroup.querySelectorAll("*").forEach((element: Element) => expect(document.body.contains(element)).toBeTruthy());
 
                 (dataView.metadata.objects as any).legend.show = false;
                 visualBuilder.updateFlushAllD3Transitions(dataView);
-                expect(visualBuilder.legendGroup.children()).not.toBeInDOM();
```
Comment (MulyukovAidar) [inline test/visualTest.ts:151]: please explain

**Review by MulyukovAidar — CHANGES_REQUESTED** [review]

File: src/settings.ts (hunk 2 above)
Comment (kullJul) [reply src/settings.ts]: fixed 3c18a8a6a8d9ace4f2b0e677d83c16800566e9de

File: src/radarChart.ts (hunk 1 above)
Comment (kullJul) [reply src/radarChart.ts]: fixed 3c18a8a6a8d9ace4f2b0e677d83c16800566e9de

File: src/radarChart.ts [hunk 5]
```diff
@@ -1153,6 +1160,11 @@ export class RadarChart implements IVisual {
 
         this.legend.drawLegend(legendData, { height, width });
         LegendModule.positionChartArea(this.svg, this.legend);
+
+        d3Select(this.root.node()).selectAll("g#legendGroup text")
```
[the remark this answers is not in the document]
Comment (kullJul) [reply src/radarChart.ts]: fixed 22e69dbdaa6a4880413d6c60aaf37fead8d9e68a

File: capabilities.json [hunk 6]
```diff
@@ -167,74 +100,41 @@
...
                         "numeric": true
                     }
                 },
                 "axisBeginning": {
-                    "displayNameKey": "Visual_AxisStartPosition",
-                    "displayName": "Axis start position",
                     "type": {
-                        "enumeration": [
-                            {
-                                "value": "-1",
```
[the remark this answers is not in the document]
Comment (kullJul) [reply capabilities.json:206]: checked, it's backward compatible

**Pull request merged**